packages/terra-paginator/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-paginator",
+  "main": "lib/Paginator.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
```

Comment:
Don't forget this description

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/c619c108cd3f92c3084a93d5dc3d99e53bd717ba